### ISSUE TITLE
Mouse acceleration/smoothing request 

### ISSUE THREAD

**Issue by frozenjo** [Lyall/GBFRelinkFix#10]
Hi, 

1st, awesome mod and thanks for releasing it. Almost refunded the game because of the lack of UW/Fov Slider.

I wanted to ask, and I know this is more of a gameplay option/request, but do you happen to know if there's a way to elimnate the mouse smoothing in Relink ?

**Comment by klownk** [comment]
Yeah, if you can add raw input for the mouse (can hope) it will be awesome.
